.gitignore

```diff
@@ -1,6 +1,7 @@
 /packages/**/*/dist/
 /examples/
 /dist
+/packages/@apphosting/publish-dev/.local
 
```

Comment:
I notice that running npm adduser --registry http://localhost:4873 generates packages/@apphosting/publish-dev/htpasswd which holds the credentials I entered. We probably want that in the .gitignore too to prevent any accidental credential leaks.